web/frontend/src/components/slam.vue

```diff
@@ -39,6 +40,21 @@ const concatArrayU8 = (arrays: Uint8Array[]) => {
   return result;
 };
 
+const fetchSLAMImageMap = (name: string) => {
+  const req = new slamApi.GetMapRequest();
+  req.setName(name);
+  req.setMimeType('image/jpeg');
+  req.setIncludeRobotMarker(true);
+  rcLogConditionally(req);
+  props.client.slamService.getMap(req, new grpc.Metadata(), (error, response) => {
+    if (error) {
+      return displayError(error);
+    }
+    const blob = new Blob([response!.getImage_asU8()], { type: 'image/jpeg' });
+    imageMap = URL.createObjectURL(blob);
+  });
+};
+
 const fetchSLAMMap = (name: string) => {
```

Comment:
Is this still used for the 3D view? Should we revert this back to GetMap as well?

Reply:
leaving this as is as fahimna is good with just not rendering the 3d view for now.